projects/ngx-formentry/src/form-entry/question-models/workspace-launcher.model.ts

```diff
@@ -0,0 +1,18 @@
+import { QuestionBase } from './question-base';
+import { AfeControlType } from '../../abstract-controls-extension/afe-control-type';
+import { WorkspaceLauncherOptions } from './interfaces/workspace-launcher-options';
+
+export class WorkspaceLauncherQuestion extends QuestionBase {
+  buttonLabel: string;
+  workspaceName: string;
+
+  constructor(options: WorkspaceLauncherOptions) {
+    console.log('constructing from ', options);
+    super(options);
+    this.renderingType = 'workspace-launcher';
+    this.label = options.label || '';
+    this.buttonLabel = options.buttonLabel || '';
+    this.workspaceName = options.workspaceName || 'order-basket';
+    this.controlType = AfeControlType.AfeFormControl;
```

Comment:
I think it's to ensure [this code does the right thing](https://github.com/openmrs/openmrs-ngx-formentry/blob/f46dfc131c23ffc08210b267436ba4e2126d853f/projects/ngx-formentry/src/form-entry/form-factory/form-control.service.ts#L46). Technically, this should probably be `AfeControlType.None` because I can't see why we need validators of value calculators on a button, but if it works, 🤷.

Reply:
Oops, I missed this comment